apps/src/dance/ai/DanceAiModal.tsx

```diff
@@ -7,27 +7,28 @@ import {useAppDispatch} from '@cdo/apps/util/reduxHooks';
 import {closeAiModal, DanceState} from '../danceRedux';
 import classNames from 'classnames';
 import {FieldDropdown, Workspace} from 'blockly/core';
-import {
-  chooseEffects,
-  ChooseEffectsQuality,
-  getGeneratedEffectScores,
-} from './DanceAiClient';
 import DanceAiScore, {ScoreColors} from './DanceAiScore';
 import AiVisualizationPreview from './AiVisualizationPreview';
-import AiBlockPreview from './AiBlockPreview';
+
 import {
   AiFieldValue,
   AiOutput,
+  DanceAiModelItem,
   DanceAiSound,
   FieldKey,
   GeneratedEffect,
   MinMax,
+  ChooseEffectsQuality,
+  Mode,
+  Toggle,
 } from './types';
 import {
+  chooseEffects,
   generateAiEffectBlocks,
   generateAiEffectBlocksFromResult,
-  generatePreviewCode,
-  getEmojiImageUrl,
+  getItem,
```

Comment:
This should likewise be renamed. `getEmoji`? `getEmojiItem`?

Reply:
I agree that `getItem` is vague. `getEmoji` seems clear and concise.

Reply:
This comment was out of date - it was already moved over to `getDanceAiModalItem`

Reply:
In general, I'm curious why we're using 'item' vs 'emoji' or 'input'.

Reply:
We had a bunch of terms in the code initially, and they haven't been totally cleaned up and standardized. But, what I've been roughly working towards:

- inputs are emoji ids
- `DanceAiModelItem` are the values loaded out of the `ai-inputs.json` file (previously just `item`)
- `emoji` is overloaded in that it could be thought of as the human readable string, the actual image, or a `DanceAiModelItem`.

Reply:
Thanks! This is helpful and makes sense.
I see we are using `selectedEmojis`. Would it make sense to call these `selectedDanceAiModalItems`?